fail2ban/tests/clientreadertestcase.py

```diff
@@ -649,8 +649,9 @@ def testReadStockJailConfForceEnabled(self):
 
 			# and we know even some of them by heart
 			for j in ['sshd', 'recidive']:
-				# by default we have 'auto' backend ATM
-				self.assertTrue(['add', j, 'auto'] in comm_commands)
+				# by default we have 'auto' backend ATM, but some distributions can overwrite it, 
+				# (e.g. fedora default is 'systemd') therefore let check it without backend...
+				self.assertTrue([True for i in comm_commands if len(i) == 3 and i[0:2] == ['add', j]])
```

Comment:
ok -- here is the results of the test:
```
$> python /tmp/time_check.py
checks
f1: True, True
f2: True, True
Last entry
f1: 0.0000450
f2: 0.0000766
First entry
f1: 0.0000444
f2: 0.0000007
```
where f2 is `any(len(x) == 3 and ["add", j] == x[:2] for x in comm_commands)` . Most efficient and compromised between our two ;)